components/AppToolbar.vue

```diff
@@ -1,9 +1,14 @@
+<!-- eslint-disable @intlify/vue-i18n/no-raw-text -->
 <template>
   <header
-    class="py-4 px-6 border-b border-slate-100 bg-white/80 backdrop-blur-md sticky top-0"
+    class="flex justify-between py-4 px-6 border-b border-slate-100 bg-white/80 backdrop-blur-md sticky top-0"
   >
     <strong v-if="toolbarTitleStore().toolbarTitle">
       {{ toolbarTitleStore().toolbarTitle }}
     </strong>
+    <light class="flex">
```

Comment:
I dont think there is a HTML element called `light` :) Did you want the text to be a light font? If so, use a span instead with the class `font-light`